Gemfile

```diff
@@ -2,8 +2,16 @@ source 'https://rubygems.org'
 
 ruby '3.0.2'
 
+gem 'sinatra'
+gem 'sinatra-contrib'
+gem 'sinatra-flash'
+gem 'webrick'
+gem 'pg'
+gem 'bcrypt'
+
```

Comment:
It says in the code review rubric that some of these should be in a 'development' group, but not sure how much of a difference it makes!